components/record/MediaPresentation.vue

```diff
@@ -97,8 +84,14 @@
           iiif: Number(process.env.ENABLE_IIIF_MEDIA)
         });
       },
-      isPDF() {
-        return isPDF(this.media);
+      imageMediaType() {
```

Comment:
I don't think it makes sense to do this here just to pass it into MediaImage.vue

Let's refactor so that:
1. MediaImage.vue takes the same props as MediaPresentation.vue, i.e. imageSrc, media and europeanaIdentifier
2. imageLink is moved into MediaImage.vue
3. isPDF and isImage are imported in MediaImage.vue, not in MediaPresentation.vue, and used to determine the i18n key directly instead of setting this additional media type property